.github/workflows/bindings-check.yml

```diff
@@ -4,7 +4,12 @@ on:
   push:
     branches: [main]
   pull_request:
-    paths: ["nitro-protocol/contracts/**", generate-adjudicator-bindings.sh]
+    paths:
+      [
+        "nitro-protocol/contracts/**",
+        generate-adjudicator-bindings.sh,
+        node/engine/chainservice/adjudicator/**,
```

Comment:
What's the reason for adding this single `chainservice/adjudicator` directory but not the other similar directories for other contracts?
```
chainservice/consensusapp
chainservice/erc20
chainservice/virtualpaymentapp
```